package.json

```diff
@@ -268,7 +268,8 @@
 	"scripts": {
 		"build": "npm run build:packages && wp-scripts build",
 		"build:analyze-bundles": "npm run build -- --webpack-bundle-analyzer",
-		"build:package-types": "node ./bin/packages/validate-typescript-version.js && ( tsc --build || ( echo 'tsc failed. Try cleaning up first: `npm run clean:package-types`'; exit 1 ) ) && node ./bin/packages/check-build-type-declaration-files.js",
+		"build:package-types": "npm run --silent build:package-types:verbose",
+		"build:package-types:verbose": "node ./bin/packages/validate-typescript-version.js && ( tsc --build || ( echo 'tsc failed. Try cleaning up first: `npm run clean:package-types`'; exit 1 ) ) && node ./bin/packages/check-build-type-declaration-files.js",
```

Comment:
Do we really need two scripts, can we just keep one by default have an option to make it verbose or silent?

Reply:
Alternatively, printing the failure could go in another script:

```sh
tsc --build || npm run print-typescript-clean-suggestion
```

Reply:
[4917090](https://github.com/WordPress/gutenberg/pull/61530/commits/49170909df9c6dc14b0eefe7807de1e5fb4ec1d5)